docs/hip_sdk_install_win/hip_sdk_install_win.md

```diff
@@ -72,48 +80,62 @@ to [Installing Components](#installing-components).
 demonstrates the installer window once the installation components are all
 deselected.
 
-| ![DeSelect All](../data/hip_sdk_install_win/DeSelectAll.png) |
-|:------------------------------------------------------------:|
-| **Figure 4. Deselect All Selection**                         |
+:::{figure} ../data/hip_sdk_install_win/DeSelectAll.png
+:name: deselect-all
+:alt: DeSelect All
+Deselect All Selection
+:::
 
 #### HIP Components
 
 By default, each HIP component will be checked off for full installation,
-Figures 4 through 8 demonstrate the options available to you when you click
+{numref}`deselect-all` through {numref}`bitcode-profiler` demonstrate the options available to you when you click
 **Additional Options** under each component.
 
-| **Table 2. Custom Selections for Installation**                   |                                                      |
-|:------------------------------------------------------------------|:---------------------------------------------------- |
+:::{table} Custom Selections for Installation
+:name: custom-selections
 | **If:**                                                           | **Then:**                                            |
+|:------------------------------------------------------------------|:---------------------------------------------------- |
 | You intend to make custom selections for this installation        | Skip to the section _Deselect All_.                  |
 | You do not intend to make custom selections for this installation | Continue to the section _AMD Display Driver_.        |
+:::
 
 **Note** You can manually select installation locations for the HIP SDK Core, as
-shown in Figure 5.
-
-| ![HIP SDK Core](../data/hip_sdk_install_win/HIP-SDK-Core.png) |
-|:-------------------------------------------------------------:|
-| **Figure 5. HIP SDK Core Option**                             |
-
-| ![HIP Libraries](../data/hip_sdk_install_win/HIP-Libraries.png) |
-|:---------------------------------------------------------------:|
-| **Figure 6. HIP Libraries Option**                              |
-
-| ![HIP Runtime Compiler](../data/hip_sdk_install_win/HIP-Runtime-Compiler.png) |
-|:-----------------------------------------------------------------------------:|
-| **Figure 7. HIP Runtime Compiler Option**                                     |
-
-| ![HIP Ray Tracing](../data/hip_sdk_install_win/HIP-Ray-Tracing.png) |
-|:-------------------------------------------------------------------:|
-| **Figure 8. HIP Ray Tracing**                                       |
-
-| ![BitCode Profiler](../data/hip_sdk_install_win/BitCode-Profiler.png) |
-|:---------------------------------------------------------------------:|
-| **Figure 9. BitCode Profiler**                                        |
+shown in {numref}`hip-sdk-core`.
+
+:::{figure} ../data/hip_sdk_install_win/HIP-SDK-Core.png
```

Comment:
We should switch to figure-md syntax for better markdown rendering.

Reply:
The HTML output is the same and they can be used interchangeably. Do we still want to switch?